src/components/Link/Link.tsx

```diff
@@ -152,6 +154,7 @@ export const Link = chakra(
     const {
       children,
       className,
+      hasVisitedState = false,
```

Comment:
This should be true by default according to the ticket, unless if there was an update?

Reply:
Oops. Great catch. I was testing. Thank you!